docs/guide/field_type_reference.md

```diff
@@ -689,18 +705,18 @@ This Field Type does not support settings.
 
 Using the variation Service, variations of the original image can be obtained. They are `\eZ\Publish\SPI\Variation\Values\ImageVariation` objects with the following properties:
 
-| Property       | Type     | Example  | Description                          |
-|----------------|----------|----------|--------------------------------------|
-| `width`*       | int      | `640`    | The variation's width in pixels      |
-| `height`*      | int      | `480`    | The variation's height in pixels     |
+| Property       | Type     | Example  | Description|
+|----------------|----------|----------|------------|
+| `width`*       | int      | `null`    | The variation's width in pixels. For more details see Caution note below.|
+| `height`*      | int      | `null`    | The variation's height in pixels. For more details see Caution note below.|
```

Comment:
Ref: 
> Properties marked with an asterisk are currently unsupported. They are available but their value is always `null`.

I'm not sure `width` and `height` of `ezimage` are not supported. AFAICS value is being set, so might not be `null`. But maybe there's a broader context here. @kmadejski @adamwojs @natanael89  You've worked with that FT more than I have, any truth to that statements about those properties?

Reply:
@alongosz unfortunately, at the moment it is always `null`. The only exception here is in case of eZ Publish 5.4.x where you can use legacy alias generator. This is related to the: https://jira.ez.no/browse/EZP-25487

Reply:
Ok, that explains it. Thanks for the clarification.